mockupdb/__init__.py

```diff
@@ -205,7 +201,7 @@ def __init__(self):
 
     def result(self, timeout=None):
         self._event.wait(timeout)
-        # wait() always returns None in Python 2.6.
+        # wait() always returns None in Python 2.
```

Comment:
Actually it returns a bool in Python 2.7. So I *believe* this could now be:
```
if not self._event.wait(timeout):
    raise AssertionError('timed out waiting for Future')
```

Reply:
Oh you're right! I was looking at the python 3 docs which only mention 3.1 not 2.7:
> Changed in version 3.1: Previously, the method always returned None.

https://docs.python.org/3/library/threading.html#threading.Event.wait
https://docs.python.org/2/library/threading.html#threading.Event.wait